apps/daimo-web/src/app/bridge/Widget.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import { LiFiWidget, WidgetConfig } from "@lifi/widget";
+import { useMemo } from "react";
+import { Address } from "viem";
+
+const widgetConfigBase: WidgetConfig = {
+  integrator: "daimo",
+  variant: "default",
+  subvariant: "default",
+  appearance: "light",
+  buildUrl: false,
+  disabledUI: ["toAddress", "toToken"],
+  hiddenUI: ["appearance", "language"],
+  slippage: 0.002,
+  containerStyle: {
+    border: "1px solid rgb(234, 234, 234)",
+    borderRadius: "16px",
+    maxHeight: "1200px",
+  },
+  languages: {
+    default: "en",
+    allow: ["en"],
+  },
+};
+
+export function Widget({
+  toName,
+  toAddress,
+}: {
+  toName: string;
+  toAddress: Address;
+}) {
+  const widgetConfig = useMemo<WidgetConfig>(
+    () => ({
+      ...widgetConfigBase,
+      languageResources: {
+        en: {
+          header: {
+            exchange: "Deposit to Daimo",
+          },
+          main: {
+            sendToWallet: `Send to ${toName} on Daimo`,
+          },
+        },
+      },
+      toAddress,
+      toChain: "bas",
+      toToken: "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
```

Comment:
if you use the import from env / @daimo/common this would be invalid on testnet deploys and only valid on mainnet